src/plone/restapi/services/aliases/add.py

```diff
@@ -94,11 +94,19 @@ def _reply_csv(self):
         form = self.request.form
         if not form.get("file"):
             raise BadRequest("No file uploaded")
+
+        file = form["file"]
+
+        if not hasattr(file, "headers"):
+            raise BadRequest("Uploaded file is not a valid file")
```

Comment:
HTTPRequest only creates a FileUpload object if there are headers (see https://github.com/zopefoundation/Zope/blob/master/src/ZPublisher/HTTPRequest.py#L549) so I don't think this check is necessary.